### PR TITLE
Devfest Uyo

### PR DESCRIPTION
**Pull request by PrimeFord**

### CODE REVIEW

**Comment by PrimeFord** [comment]
bruhhh I dont know why the checks are failing

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ cd ng-tech-events
 - Set the remote upstream of your forked repository to the base repository (i.e. the original repository)
 
 ```shell
-    git remote add upstream https://github.com/kaf-lamed-beyt/ng-tech-event.git
+    git remote add upstream https://github.com/kaf-lamed-beyt/ng-tech-events.git
```
Comment (kaf-lamed-beyt) [inline CONTRIBUTING.md:34]: Ayy! Than you for this @PrimeFord! 🚀

**Review by kaf-lamed-beyt — APPROVED** [review]
The builds failing are most likely from the last push we made. I'll fix

Thanks again chief!

**Comment by kaf-lamed-beyt** [comment]
Fixes #88

**Pull request merged**